csharp_dotnetcore/60.ConversationHistory/TranscriptStore.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Bot.Builder.Azure;
+
```

Comment:
copyright headers.

Reply:
Not applicable anymore.

---
In reply to: [218236617](https://github.com/Microsoft/BotBuilder-Samples/pull/391#discussion_r218236617) [](ancestors = 218236617)